README.md

```diff
@@ -1,93 +1,284 @@
 
-For federated and incubating projects, the Project Team decides among themselves on how to add and/or remove members from the Project Team. Further, Project Teams for incubating projects are exempted from the requirement to have at least one Steering Council member. For a community maintained project, a petition for commit rights is made to the current members of the Project Team for that project. If this petition passes according to the voting rules for "Nominate new member of a Project Team" below, the petitioner is granted write access and added to the team.
+In the other example, Microsoft is a large company and these three largely unrelated parts of it are unlikely to have common goals based mainly on their common ultimate funder. However, a company that wanted to control the conda Organization could do so by enlisting people from across their organization, claim separate funding, and then capture the council. **In this case, the default guideline is to treat all parts of a company as having one funder.**
 
-**Steering Council** members are excluded from these votes and retain commit rights to all community projects as stated above. They can add or remove themselves from Project Teams as they see fit in order to indicate which projects they are involved with, have a say in voting for write access petitions, etc. Steering Council members may not add themselves to a Project Team in order to vote on a petition for commit access if they were not already on that team when the petition was made.
+#### Tracking Funding
 
+Nominated members will provide a list of organizations that fund above 25% of their time. Nominees will need to provide a complete “chain of funding” for each funding source. For example: “GitHub, a part of Microsoft”, or “CS Dept at Western Michigan University, *Computational Reproducibility Made Easy grant*, NHGRI, NIH, US Federal Government.”
+
+The Steering Council determines the appropriate funding level(s) to consider when looking for common funding with existing and proposed members. The funding source of any Steering Council member can be put to a vote under the policy for [All Other Items](#all-other-items) below.
+
+Council members need to keep their funding documentation up to date, and notify the council whenever it changes. Should a change in funding state create a shared funding situation, the council members should decide who among themselves should move to emeritus status. If no agreement can be reached, the Steering Council will hold a simple vote to determine which member is moved to emeritus status under the [Resolve Overlapping Funding](#resolve-overlapping-funding) voting provision below.
```

Comment:
What is the enforcement mechanism for failure to comply or keep records up to date? What is the time period for funding updates?

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.

Reply:
The council always has the power to kick people off the council.  We could state that here, but it is covered elsewhere.

I am not opposed to restating that here.

I actually lean against specifying a time line here.  We could say something like "Within two weeks." Or, in practice, we could say "Hey! Update your status!"  If someone does not, remind them in two weeks.  If they still don't then find out what is going on.  Do we need to codify that?

Also see my general comment in response to your general comment.

Reply:
@CJ-Wright: The text has been updated with a specific deadline for updating funding information, and specified action to take if that deadline is missed.